assets/src/dashboard/app/views/templateDetails/components.js

```diff
@@ -90,6 +90,7 @@ export const Text = styled(Paragraph2)`
 `;
 
 export const MetadataContainer = styled.fieldset`
+  border: 0;
```

Comment:
@carlos-kelly this is where that border's coming from in storybook. Common css is zeroing this out on its own, but that's not in storybook. I think i might look into adding common css to our storybook set up in the dashboard so we have more consistency, but i'd rather keep that separate.

Reply:
common as in common.css loaded by WordPress? Yeah let's please not add this. Our components should work standalone :-)